src/Command/BuyCommand.php

```diff
@@ -56,6 +57,18 @@ public function execute(InputInterface $input, OutputInterface $output): int
             return 1;
         }
 
+        $assetToBuy = (string) $input->getArgument('asset');
+        // TODO check if this a valid token which can be bought on the exchange
```

Comment:
Goed idee, een publieke methode in de BuyServiceInterface zou een goede plek zijn.

Reply:
Is toegevoegd in de BuyServiceInterface en de TODO's staan ook in de <Exchange>BuyService files